### DISCUSSION TITLE
Should we change the prediction value of Anomaly Detectors from integers (1, 0) to booleans (true, false)?

### DISCUSSION THREAD

**Discussion by andrewdalpino** [RubixML/ML#257] [Ideas]
In line with the whole "strings" are discrete, integers and floating point numbers are continuous rule, it may be more natural to return booleans for anomaly predictions than an integer. There would be no memory savings as both require 64 bits each. Thoughts?

**Comment by DrDub** [comment]
OTOH, the 0,1 allows for changing them to a floating point down the road...
